lib/pf/node.pm

```diff
@@ -766,7 +766,17 @@ sub nodes_maintenance {
     while (my $row = $iter->next()) {
         my $currentMac = $row->{mac};
         pf::dal->set_tenant($row->{tenant_id});
+
+        my $apiclient = pf::client::getClient;
+        my %violation_data = (
+            'mac'   => $currentMac,
+            'tid'   => 'node_maintenance',
+            'type'  => 'internal',
+        );
+        $apiclient->call('trigger_violation', %violation_data);
```

Comment:
`trigger_violation` and `%violation_data` need to be renamed to `security_event`.